torch/optim/__init__.py

```diff
@@ -17,15 +17,3 @@
 from .optimizer import Optimizer
 from .lbfgs import LBFGS
 from . import lr_scheduler
-
-del adadelta
```

Comment:
why are you removing these? we need these deleted so that they dont show up in autocomplete and confuse users.

Reply:
@soumith I kinda just wanted to see what would happen (can I run through the tests without anyone getting notifications about my pull requests?).

E.g. changing `del DoubleStorageBase` to `del _C.DoubleStorageBase` resulted in build failures in jenkins:

`
Dec 03 19:51:03 __________________ ERROR collecting test/onnx/test_models.py ___________________
Dec 03 19:51:03 /var/lib/jenkins/workspace/test/onnx/test_models.py:1: in <module>
Dec 03 19:51:03     from torchvision.models.alexnet import alexnet
Dec 03 19:51:03 /var/lib/jenkins/.local/lib/python2.7/site-packages/torchvision/__init__.py:1: in <module>
Dec 03 19:51:03     from torchvision import models
Dec 03 19:51:03 /var/lib/jenkins/.local/lib/python2.7/site-packages/torchvision/models/__init__.py:1: in <module>
Dec 03 19:51:03     from .alexnet import *
Dec 03 19:51:03 /var/lib/jenkins/.local/lib/python2.7/site-packages/torchvision/models/alexnet.py:1: in <module>
Dec 03 19:51:03     import torch.nn as nn
Dec 03 19:51:03 /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/__init__.py:301: in <module>
Dec 03 19:51:03     from . import _torch_docs, _tensor_docs, _storage_docs
Dec 03 19:51:03 /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/_storage_docs.py:44: in <module>
Dec 03 19:51:03     """)
Dec 03 19:51:03 /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/_storage_docs.py:20: in add_docstr_all
Dec 03 19:51:03     cls = getattr(torch._C, cls_name)
Dec 03 19:51:03 E   AttributeError: 'module' object has no attribute 'DoubleStorageBase'
`

I'm not entirely clear what it is or isn't deleting. flake8 says we're trying to delete something that's undefined, and as you can see when I delete it in a way flake8 recognizes it seems like we actually needed it.

I can also try:
import torch
del torch.optim.adadelta
....
but I'm not sure if that would just give me errors like the above.

Reply:
@nairbv If you prepend [please ignore] to the front of your PR or [wip] fewer people will look at it